### DISCUSSION TITLE
Tailwind doesn't style my page when I enable jit mode

### DISCUSSION THREAD

**Discussion by k4l3b4** [tailwindlabs/tailwindcss#5531] [Help]
I started a React project from scratch(without create-react-app) after configuring tailwindcss all seems to work fine but when I enable jit mode my page reverts to an unstyled plain HTML page.

**What version of Tailwind CSS are you using?**
2.2.6

**What build tool (or framework if it abstracts the build tool) are you using?**
webpack 5.53.0, postcss 8.3.6

**What version of Node.js are you using?**
14.17.0

**What browser are you using?**
Chrome(Brave)

**What operating system are you using?**
Windows

**Reproduction repository**
https://github.com/k4l3b-plo/tailwind-myissue

**Comment by aggreggator** [comment]
Please recheck your second purge entry:
https://github.com/k4l3b-plo/tailwind-myissue/blob/main/tailwind.config.js#L4

**Reply by k4l3b4** [reply]
i tried both `purge: ['./src/**/*.{js,jsx,ts,tsx,html}'],` and `purge: ['./src/**/*.{js,jsx,ts,tsx}', './src/template/index.html']`, but still not working

**Comment by simonswiss** [comment]
Hey!

I tried to pull your code, and started the server, and it seems to work fine with `jit` mode turned on?

![image](https://user-images.githubusercontent.com/485747/134124599-e26b08a7-36cc-49f2-afa1-f4b5ac2bff2c.png)

It also supports arbitrary values as expected:

![image](https://user-images.githubusercontent.com/485747/134124722-3ac8a120-b8c0-4062-8620-7520dbe79310.png)

After adding the `src` segment in your purge config, the dark classes applied to the `template/index.html` also work properly.

```diff
- purge: ["./src/**/*.{js,jsx,ts,tsx}", "./template/index.html"],
+ purge: ["./src/**/*.{js,jsx,ts,tsx}", "./src/template/index.html"],
```

![image](https://user-images.githubusercontent.com/485747/134125740-2f068d6b-6ae2-4e35-8172-6af8e8a08fbd.png)

Mind you, I am not on Windows but on Mac OS – but it works properly over here, in your reproduction repo.

**Reply by k4l3b4** [reply]
do you have an idea of what might be the problem? I pulled the repo and tried it but still the same
